lib/hanami/validations/form.rb

```diff
@@ -14,7 +14,18 @@ module Validations
     #
     #   class Signup
     #     include Hanami::Validations::Form
+    #
+    #     validations do
+    #       required(:name).filled(:foo?)
+    #       optional(:location).maybe(:str?)
```

Comment:
Because of the `#maybe` here, the following would pass... Is it worth mentionning?
```ruby
assert Signup.new(name: 'Luca', location: nil).validate.success?
```